config/database.yml.example

```diff
@@ -0,0 +1,20 @@
+development: &dev
```

Comment:
👏👏👏 
The `devsdeck` database name makes sense, for sure! =)

There are just some points that I would like to address.

Rails itself supports the presence of a `ENV['DATABASE_URL']` variable, what pushes Heroku (and I'm pretty sure others as well) into doing things like the one explained in [here](https://devcenter.heroku.com/articles/rails-database-connection-behavior), and I quote:

>While this might work locally, Heroku runs a Postgres server on a different machine, so it 
>would not work if your app tried to connect to localhost with a blank password when you 
>deployed to Heroku. By always connecting to ENV['DATABASE_URL'] when it is present, 
>such problems are non-existent.

I'm bring this to the table because I think we better go with a REALLY minimalist/simple `database.yml.example` (the configuration in the one that was already on `database.yml` is a default that *just works* ™). And let each developer decide about the specifics.
If we REALLY feel the need to enforce some kind of initial configuration, in this case I would go for something like the usage of `url: <%= ENV['DATABASE_URL'] %>` on the `:default` configuration.

wdyt ❓